go/billing/tests/test_admin.py

```diff
@@ -72,3 +75,20 @@ def test_account_admin_view(self):
         self.assertContains(response, 'Description')
         self.assertContains(response, 'Credit balance')
         self.assertContains(response, 'Last topup balance')
+        self.assertContains(response, 'Is developer')
+
```

Comment:
Ah, I see what is actually missing are tests for the ``set_developer_flag`` and ``unset_developer_flag`` actions?